plugins/synaptics-rmi/fu-synaptics-rmi-device.c

```diff
@@ -578,9 +578,9 @@ fu_synaptics_rmi_device_setup (FuDevice *device, GError **error)
 				  f01_basic->data[2],
 				  f01_basic->data[3],
 				  priv->flash.build_id);
-	fu_device_set_version (device, fw_ver, FWUPD_VERSION_FORMAT_TRIPLET);
+	fu_device_set_version (device, fw_ver);
 	bl_ver = g_strdup_printf ("%u.0", priv->flash.bootloader_id[1]);
-	fu_device_set_version_bootloader (device, bl_ver, FWUPD_VERSION_FORMAT_TRIPLET);
+	fu_device_set_version_bootloader (device, bl_ver);
```

Comment:
This is emitting `54.0 is not a valid triplet`....

Reply:
Good catch. This needs to be 0.0.%u I guess.

Reply:
Fixed.